model/implicit_constraints.py

```diff
@@ -47,6 +51,24 @@ def __init__(self, model, var, data):
         self.add_isolated_off_days(var.gamma, var.q)
         self.add_consecutive_days(var.gamma, var.q)
 
+    """
+    def add_no_shift_while_no_demand(self, x):
+
+        self.model.addConstrs(
+            (
+                quicksum(
+                    quicksum(
+                        x[e, t_marked, v] for t_marked in self.combined_time_periods
+                        if t_marked + v - self.time_step in self.time_periods_with_no_demand
+                    )
+                    for v in self.shift_durations["work"]
+                ) == 0
+                for e in self.employees
+            ),
+            name="no_work_shift_ending_in_no_demand_allowed"
+        )
+    """
+
```

Comment:
Should this be deleted @sondrenaerland ?

Reply:
Jess, den kan fjernes!